task/github.py

```diff
@@ -423,6 +423,19 @@ def issues(
             result += issues
         return result
 
+    def issue_comments(self, number: int) -> Sequence[JsonObject]:
```

Comment:
Hmm, I wonder if this belongs into task/__init__.py rather? That has the higher-level abstractions, while github.py is the "basic" API. The separation isn't totally clear-cut of course.